src/Xamarin.Android.Build.Tasks/Xamarin.Android.Legacy.targets

```diff
@@ -223,7 +223,7 @@ projects. .NET 5 projects will not import this file.
       />
       <FilteredAssemblies
           Include="@(ReferenceCopyLocalPaths)"
-          Condition="'%(ReferenceCopyLocalPaths.ResolvedFrom)' != 'ImplicitlyExpandDesignTimeFacades' And '%(ReferenceCopyLocalPaths.Extension)' == '.dll' And '%(ReferenceCopyLocalPaths.RelativeDir)' == '' And Exists('%(ReferenceCopyLocalPaths.Identity)') "
+          Condition="'%(ReferenceCopyLocalPaths.ResolvedFrom)' != 'ImplicitlyExpandDesignTimeFacades' And '%(ReferenceCopyLocalPaths.Extension)' == '.dll' And Exists('%(ReferenceCopyLocalPaths.Identity)') "
```

Comment:
In the original commit that added the `%(ReferenceCopyLocalPaths.RelativeDir)' == ''` condition (https://github.com/xamarin/xamarin-android/commit/8f2ae24813ec3cdab56f688c4c3b611a33759333), it sounded like the condition wasn't a critical part of the functionality. Additionally, my understanding is that the mechanism for resolving the proper runtime assembly when given a reference assembly (such as for NuGet packages) has also been improved within `<ResolveAssemblies/>` since that time. So I believe that this condition can be removed without undesired consequences.

Reply:
Yeah, I don't see why that condition would be there...